sacrebleu/dataset/wmt_xml.py

```diff
@@ -82,20 +87,29 @@ def get_sents(doc):
                 ref_doc.get("translator"): get_sents(ref_doc) for ref_doc in ref_docs
             }
 
+            hyp_docs = doc.findall(".//hyp")
+            hyps = {
+                hyp_doc.get("system"): get_sents(hyp_doc) for hyp_doc in hyp_docs
+            }
+            # print("SRC", len(src_sents))
+            # print("HYP", hyps.keys(), [len(list(hyps[key])) for key in hyps.keys()])
+
             for seg_id in sorted(src_sents.keys()):
-                # no ref translation is avaliable for this segment
+                # no ref translation is avalaible for this segment
```

Comment:
A colleague of mine is training a Grammar Error Correction models based on such edits:-)

Reply:
Good idea :)

Reply:
Well, not such a good idea in practice, e.g. when a typo is changed to another typo as in this case.

Reply:
:)